apps/teams-test-app/src/components/PagesAPIs.tsx

```diff
@@ -85,7 +85,7 @@ const NavigateToApp = (): React.ReactElement =>
 
 const ShareDeepLink = (): ReactElement =>
   ApiWithTextInput<DeepLinkParameters & ShareDeepLinkParameters>({
-    name: 'core.shareDeepLink',
+    name: 'pages.shareDeepLink',
```

Comment:
I'm curious if the e2e tests will fail with this change (unless you already updated them in the corresponding host sdk repo).